packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -653,11 +654,6 @@ export default class TradeStore extends BaseStore {
     @action.bound
     onUnmount() {
         this.disposeSwitchAccount();
-        this.proposal_info = {};
-        this.purchase_info = {};
```

Comment:
Why have we removed these resetters? Is it due to the upcoming changes?

Reply:
Because we're already resetting them using the `refresh` method `onMount`.